.github/workflows/python-pipeline.yml

```diff
@@ -36,8 +36,8 @@ jobs:
         # TODO remove this once https://github.com/AIM-Harvard/pyradiomics/pull/761 lands
         run: |
           poetry run pip install --upgrade pip
-          poetry run pip install numpy
           poetry install
+          poetry run pip install "pyradiomics>=3.0.1"
```

Comment:
One thing I just realised is we got this install for the pipeline now. The previous numpy install was done prior because the poetry install wasn't installing it appropriately, but now we install pyradiomics after the poetry install. Is there a way we can unify this outside of this pipeline in one step for poetry?